CHANGES.rst

```diff
@@ -5,6 +5,7 @@ Version 3.1.0
 
 Unreleased
 
+-   Fix traceback-matching tests with Python 3.11.0a1 :issue:`1526`
```

Comment:
Is this PR only fixing tests, or also broken Jinja functionality. In case it's the former I don't think a changelog entry is useful (as a user I do not care about infrastructure changes)

Reply:
it only fixes tests. Honestly, I would not add a changelog entry myself, but the PR checklist kinda scared me I had to. Should I drop it?

Reply:
yeah i'd skip it. the checklist is simply very generic and covers all kinds of contributions ;)

Reply:
done